test-project-vite/package.json

```diff
@@ -54,6 +66,11 @@
     "prettier": "^2.5.1",
     "typescript": "^4.5.4"
   },
+  "//": "TODO: app-vite provides an early version of Vite than vitest",
```

Comment:
This should be fixed into core, and we'll update Vitest requirements for app-vite when the new version is released